app/src/main/java/com/example/mylittlelibrary/ui/activities/MainActivity.kt

```diff
@@ -42,27 +42,27 @@ class MainActivity : AppCompatActivity() {
             intent.putExtra("Clicked", "DVDs")
             startActivity(intent)
 
-        imageView = findViewById(R.id.imageView)
-        button = findViewById(R.id.btnAddItem)
+            imageView = findViewById(R.id.imageView)
+            button = findViewById(R.id.btnAddItem)
 
-        button.setOnClickListener{
-            val takePictureIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
+            button.setOnClickListener{
+                val takePictureIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
 
-            try {
-                startActivityForResult(takePictureIntent, REQUEST_IMAGE_CAPTURE)
-            } catch (e:ActivityNotFoundException){
-                // Toast.makeText(this,"Error: " + e.LocalizedMessage,Toast.LENGTH_SHORT).show()
-                Toast.makeText(this,"Error",Toast.LENGTH_SHORT).show()
+                try {
+                    startActivityForResult(takePictureIntent, REQUEST_IMAGE_CAPTURE)
+                } catch (e:ActivityNotFoundException){
+                    // Toast.makeText(this,"Error: " + e.LocalizedMessage,Toast.LENGTH_SHORT).show()
+                    Toast.makeText(this,"Error",Toast.LENGTH_SHORT).show()
```

Comment:
Log the error here.
Toasts with an error message can do one of two things to a user:
- Confuse the user.
- Arm a hacker.